mergify_engine/eventlogs.py

```diff
@@ -150,6 +152,13 @@ class EventUpdate(EventBaseNoMetadata):
     event: typing.Literal["action.update"]
 
 
+def _get_repository_key(
+    owner_id: github_types.GitHubAccountIdType,
+    repo_id: github_types.GitHubRepositoryIdType,
+) -> str:
+    return f"eventlogs-repository/{owner_id}/{repo_id}"
```

Comment:
```suggestion
    return f"eventlogs/repository/{owner_id}/{repo_id}"
```

Reply:
This will conflict with `events/{owner_id}/{repo_id}/{pull}`

Reply:
too sad not having added `/pull/` there I guess 😢